### DISCUSSION TITLE
Image quality issue if converted Png to Jpg

### DISCUSSION THREAD

**Discussion by ray-kudos** [dlemstra/Magick.NET#1553] [Help]
### Magick.NET version

Q8-AnyAPU 13.5.0

### Environment (Operating system, version and so on)

Windows 10 Pro 20H2

### Description

Hi,

I am trying to convert a png format image(11554_original.png) to jpg format(11554.jpg) and found out the jpg image is broken.
11554.png was OK, but 11554.jpg wasn't properly converted. I was wondering what I missed here.

![11554_original](https://github.com/dlemstra/Magick.NET/assets/147108836/3bc91ab9-f36e-4b74-99fb-cd829475e8eb)
![11554](https://github.com/dlemstra/Magick.NET/assets/147108836/e1d49f6f-1e52-4ea8-9576-a5368d22df9e)

### Steps to Reproduce

1. Save as png
string fullFileName = "c:\\temp\\11554.png";
using (var image = new MagickImage(bits))
{
	image.Format = MagickFormat.Png;
	File.WriteAllBytes(fullFileName, image.ToByteArray());
}

2. Save as jpg
string fullFileName = "c:\\temp\\11554.jpg";
using (var image = new MagickImage(bits))
{
	image.Format = MagickFormat.Jpg;
	File.WriteAllBytes(fullFileName, image.ToByteArray());				
}

**Comment by dlemstra** [comment]
A jpeg image has no transparency and when your png file is written the alpha channel is disabled and the pixels behind it are shown. I would advise you to specify the background color with the following method:

```C#
using var image = new MagickImage(@"c:\temp\11554.png");
image.ColorAlpha(MagickColors.White);
image.Write(@"c:\temp\11554.jpg");
```
